helm/hpcc/templates/roxie.yaml

```diff
@@ -109,6 +109,9 @@ spec:
 {{- end }}
       annotations:
         checksum/config: {{ $topoconfigSHA }}
+{{- if hasKey $.Values.global "metrics" }}
+{{ include "hpcc.addPrometheusScrapeAnnotations" $.Values.global.metrics | indent 8 }}
```

Comment:
trivial: we're trying to indent the template blocks (1 space) for clarity, e.g.:
```
{{- if hasKey $.Values.global "metrics" }}
 {{- include "hpcc.addPrometheusScrapeAnnotations" $.Values.global.metrics | nindent 8 }}
{{- end }}
```

Reply:
Indeed. I attempted to create a dedicated networkpolicy per pod which declares itself prometheusReporter. However it seems (for now) per component configuration isn't allowed (per schema)